src/etc/one-context.d/loc-15-keepalived##vr

```diff
@@ -160,6 +176,7 @@ vrrp_instance $interface {
   $(gen_auth)
   $(gen_track)
   $(gen_preempt)
+  $(gen_notify_script)
```

Comment:
Will require a [documentation entry](https://docs.opennebula.io/6.6/marketplace/appliances/vnf.html#vnf-keepalived-context-param) to showcase the variable VROUTER_KEEPALIVED_NOTIFY_SCRIPT